src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/ProjectServices/VsCoreProjectSystemReferenceReader.cs

```diff
@@ -40,6 +43,10 @@ internal class VsCoreProjectSystemReferenceReader
 
             _vsProjectAdapter = vsProjectAdapter;
             _threadingService = threadingService;
+
+            _referenceMetadata = Array.CreateInstance(typeof(string), 1);
```

Comment:
This can be just `new string [] { "ReferenceOutputAssembly" }`.

Reply:
Fixed.